flutter-mentorship-t4/mentorship_project
# Feature (Sign In) ✨Finishing most of Sign in logic

**Jira Ticket :** DIVA-11(https://nz-dev.atlassian.net/browse/DIVA-11)
**Description � :**
Complelte Coding of most Logic of Sign In Screen With connection With Firebase With Android &IOS

**The following sections have been fully integrated:**
1- modifying Signin Screen First
2- Running email &Password Validation
3- navigation to Home After Sign in Success if user is Logged in 
4-save user loggin and open to Home if user is Logged in 
4- Display a success message upon successful Logging
5- Show error messages if the email format is incorrect or the password .
**How to Test 📋:**
. Open the app and navigate to the SignUp screen on various iPhone devices.
. Verify that all elements are correctly positioned And Diong What it have to do.
. Open the app and navigate to the SignUp screen and make a registeration .
. Then it will navigate you Directly to Login Screen and logged in.
Screenshots/ Screen Recording 📸 :
**Before:**
![Android Emulator - Pixel_5_API_29_5554 10_3_2024 2_59_39 PM](https://github.com/user-attachments/assets/074f2208-f7d6-4684-ad8a-de06771023d2)
**After :** 
i can't provide after because i have an error and i dosnot exist in code i think it with connection to firebase
